catalog.html

```diff
@@ -3,25 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-   
-    <title>Каталог</title>
+    <title>HTML Academy: Каталог</title>
+    <link rel="icon" href="favicon.ico">
+    <link rel="icon" href="favicon/icon.svg" type="image/svg+xml">
+    <link rel="apple-touch-icon" href="favicon/180.png">
 </head>
+
 <body>
-    <header> 
+    <header>
+        <nav>
+          <a href="index.html"><!--Logo-->
+            <img src="img/nerds-logo.png" width="160" height="65" alt="Логотип">
```

Comment:
```suggestion
            <img src="img/nerds-logo.svg" width="160" height="65" alt="Логотип">
```
Логотип нужно извлечь в формате SVG.